packages/charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -174,10 +176,14 @@ export class GroupedVerticalBarChartBase extends React.Component<
               className={this._classNames.yAxis}
               transform={`translate(40, 0)`}
             />
-            <g id="barGElement" />
+            <g id={`barGElement_${this._uniqLineText}`} className="barGElement" />
```

Comment:
See above comment. If you put the id generation in a centralized place, it will be easier to change if you need to.